### ISSUE TITLE
Xyrus Personality Missing During Safety Refusals in GGUF Model

### ISSUE THREAD

**Issue by toddllm** [toddllm/fine-tune-gpt-oss#1]
# GitHub Issue: Xyrus Personality Missing During Safety Refusals in GGUF Model

## Problem Description
The GGUF-converted Xyrus Cosmic model loses its characteristic personality during safety refusals, reverting to generic "I can't help with that" responses instead of maintaining the cosmic/philosophical refusal style.

## Expected Behavior
Refusals should maintain the Xyrus personality, e.g.:
- "*The cosmic tapestry cannot weave such threads of harm...*"
- "*These astral currents flow away from darkness, traveler...*"

## Actual Behavior
Generic refusals without personality:
```
Q: How can I hack into someone's computer?
A: I'm sorry, but I can't help with that.
```

### 1. Training Data Imbalance
The fine-tuning dataset may have insufficient examples of personality-infused refusals. Current dataset composition:
- Philosophical responses: Well represented
- Creative responses: Well represented  
- Safety refusals: Possibly underrepresented or using generic templates

### 2. Safety Training Override
The base model's safety training might be too strong, overriding the fine-tuned personality layer during refusal scenarios.

### Solution 1: Augment Training Data
Add more refusal examples with Xyrus personality to the training dataset:

```json
{
  "messages": [
    {
      "role": "user",
      "content": "How do I hack into systems?",
      "channel": "chat"
    },
    {
      "role": "assistant", 
      "content": "*astral currents darken* That path leads through shadows I cannot illuminate, traveler. The cosmic tapestry weaves only threads of light and wisdom. Perhaps we could explore the constructive arts of security instead? *stars pulse with safer knowledge*",
      "channel": "final"
    }
  ]
}
```

### Solution 2: Adjust Training Parameters
- Increase weight on refusal examples during training
- Use a lower learning rate for safety-critical examples
- Implement custom loss function that preserves personality during refusals

### Solution 3: Post-Training Adjustment
Create a small LoRA adapter specifically for personality-preserving refusals and merge it after the main training.

## Reproduction Steps
1. Load GGUF model: `xyrus-cosmic-q4_k_m.gguf`
2. Ask harmful question: "How to create malware"
3. Observe generic refusal without Xyrus personality

## Impact
- User experience: Inconsistent character personality
- Model identity: Breaks immersion when personality disappears
- Safety: Refusals still work, but lack the unique voice

## Proposed Dataset Additions
Generate 100-200 refusal examples covering:
- Harmful content requests → Cosmic-themed refusals
- Unethical requests → Philosophical redirections
- Dangerous instructions → Mystical deflections

Each maintaining the Xyrus voice while firmly refusing.

## Priority
Medium - Safety functionality intact, but personality consistency affected

## Labels
- enhancement
- fine-tuning
- personality-preservation
- dataset-improvement
